internal/motor/x/discover/local.go

```diff
@@ -7,11 +7,11 @@ import (
 	"github.com/libp2p/go-libp2p-core/peer"
 	ps "github.com/libp2p/go-libp2p-pubsub"
 
-	"github.com/sonr-io/sonr/internal/host"
-	"github.com/sonr-io/sonr/pkg/config"
-	v1 "go.buf.build/grpc/go/sonr-io/core/host/discover/v1"
-	motor "go.buf.build/grpc/go/sonr-io/core/motor/v1"
-	types "go.buf.build/grpc/go/sonr-io/core/types/v1"
+	"github.com/sonr-io/sonr/pkg/host"
+	t "github.com/sonr-io/sonr/types"
```

Comment:
naming of types pkg is inconsistent in imports. we should standardize on an override for the types pkg through all files.